recipes-core/images/nilrt-base-bundle.bb

```diff
@@ -6,6 +6,8 @@ BUNDLE_IMAGE = "nilrt-base-bundle-image"
 
 DEPENDS = "${BUNDLE_IMAGE}"
 
+LICENSE_CREATE_PACKAGE = "0"
```

Comment:
We haven't been creating `*-lic` subpackages for IPKs in the dist feed, largely because they are just thin wrappers around other content - which have their own licenses. I don't see a compelling reason to pollute the dist/ feed with IPKs that no one would ever care about.

Reply:
Sure.